L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -403,6 +403,12 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
       m_l1GtMenu->print(std::cout, printV);
 
     m_l1GtMenuCacheID = l1GtMenuCacheID;
+
+    //for getting model version to condition class via GlobalBoard.runGTL, comes from menu rather than config
+    //if modelversion is uninitialized, then set it. otherwise do not change it
+    if (m_AXOL1TLModelVersion.empty()) {
+      m_AXOL1TLModelVersion = gtParser.AXOL1TLModelVersion();
+    }
```

Comment:
This is probably overcomplicated, it can probably just be `m_AXOL1TLModelVersion = gtParser.AXOL1TLModelVersion();`

Reply:
At the most, this would need to change on a run transition, but this would require a slightly more invasive change to the `L1TGlobalProducer` adding a `beginRun` function.

Reply:
following @missirol's suggestion, this has been changed to:

```
m_AXOL1TLModelVersion = ""; 
...
m_AXOL1TLModelVersion = gtParser.AXOL1TLModelVersion();
```

Loading the model once per event/condition is a deeper issue related to the core emulator class cms-dist dependencies, so that is an issue that is planned to be resolved in the near future in a dedicated PR.